src/Service.Tests/CosmosTests/MutationTests.cs

```diff
@@ -242,6 +243,49 @@ public async Task MutationMissingRequiredPartitionKeyValueReturnError()
             Assert.AreEqual("The argument `_partitionKeyValue` is required.", response[0].GetProperty("message").ToString());
         }
 
+        /// <summary>
+        /// Mutation can be performed on the authorized fields
```

Comment:
```suggestion
        /// Mutation can be performed on the authorized fields because the
        /// field `id` is an included field for the create operation on the anonymous role defined
        /// for entity 'earth'
```

Reply:
Updated